src/Nethermind/Nethermind.AuRa/Nethermind.AuRa.csproj

```diff
@@ -11,4 +11,8 @@
       <ProjectReference Include="..\Nethermind.Mining\Nethermind.Mining.csproj" />
     </ItemGroup>
 
+    <ItemGroup>
+      <PackageReference Include="Nito.Collections.Deque" Version="1.0.4" />
```

Comment:
was necessary? double checking since every related package is additional resolutions, risks, publish size

Reply:
I needed a combination of Stack and Queue (so adding from both ends). I used it to keep a sorted collection (I only added larger or smaller than all other items). But I need it to be cyclical to use less space. I almost implemented it myself but found good free solutions on good licenses (MIT).